spec/package-spec.coffee

```diff
@@ -205,3 +205,28 @@ describe "Package", ->
 
     it "uses the package name defined in package.json", ->
       expect(metadata.name).toBe 'package-with-a-totally-different-name'
+
+  describe "the initialize() hook", ->
+    it "gets called when the package is activated", ->
+      packagePath = atom.project.getDirectories()[0].resolve('packages/package-with-deserializers')
+      pack = buildPackage(packagePath)
+      spyOn(pack, 'initializeIfNeeded').andCallThrough()
```

Comment:
Now that you're spying on the main module's `initialize` method, I think you should remove this spy on `initializeIfNeeded`, because it's an implementation detail.